eox_tenant/api/v1/viewsets.py

```diff
@@ -1,29 +1,451 @@
+                    },
+                ],
+                "EDNX_USE_SIGNAL": true,
+                "EOX_CORE_UPDATE_USERS_ENABLE": true,
+                ...
+            }
+            "key": "xAer5z6FEbW",
+            "subdomain": "testing.site.edu.io",
+            "organizations": [
+                1
+            ]
+        }
+
+    **Example Create Request**:
+
+    `POST <domain>/eox-tenant/api/v1/microsites/`
+
+        body: {
+            "values": {
+                "EOX_CORE_USER_UPDATE_SAFE_FIELDS": [
+                    "is_active",
+                    "password",
+                    "fullname",
+                    "mailing_address",
+                    "year_of_birth",
+                ],
+                "EOX_HOOKS_DEFINITIONS": {
+                    "post_certificate_creation": {
+                        "action": "post_to_webhook_url",
+                        "config": {
+                            "send_certificate_data": true,
+                            "url": "https://hooks.zapier.com/hooks/catch/9477617/onlv1rq/silent/"
+                        },
+                        "fail_silently": true,
+                        "module": "eox_hooks.actions"
+                    }
+                },
+                "LANGUAGE_CODE": "en",
+                "PLATFORM_NAME": "Site TEST 1",
+                "USE_EDNX_SIGNAL": true,
+            },
+            "key": "xAer5z6FEbW",
+            "subdomain": "testing.site2.edu.io",
+        }
+
+        Response: {
+            "id": 300,
+            "values": {
+                "EOX_CORE_USER_UPDATE_SAFE_FIELDS": [
+                    "is_active",
+                    "password",
+                    "fullname",
+                    "mailing_address",
+                    "year_of_birth",
+                ],
+                "EOX_HOOKS_DEFINITIONS": {
+                    "post_certificate_creation": {
+                        "action": "post_to_webhook_url",
+                        "config": {
+                            "send_certificate_data": true,
+                            "url": "https://hooks.zapier.com/hooks/catch/9477617/onlv1rq/silent/"
+                        },
+                        "fail_silently": true,
+                        "module": "eox_hooks.actions"
+                    }
+                },
+                "LANGUAGE_CODE": "en",
+                "PLATFORM_NAME": "Site TEST 1",
+                "USE_EDNX_SIGNAL": true,
+            },
+            "key": "xAer5z6FEbW",
+            "subdomain": "testing.site2.edu.io",
+            "organizations": [
+                2
+            ]
+        }
+
+    **Example Update Request**:
+
+    `PATCH <domain>/eox-tenant/api/v1/microsites/xAer5z6FEbW/ (Sending microsite key)`
 
-class MicrositeViewSet(viewsets.ModelViewSet):
-    """MicrositeViewSet that allows the basic API actions."""
+    `PATCH <domain>/eox-tenant/api/v1/microsites/1001/ (Sending microsite id)`
```

Comment:
We can remove the example using the id. We don't want anyone using it. Let's not encourage it in the docs.

Reply:
Done